sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,8 +8,10 @@
 Removed `disableAuthoriyValidaionSafetyCheck` for GA, will reintroduce in next beta.
 
 ### Bugs Fixed
+Correctly use an `AppServiceMsiCredential` in the case both `IDENTITY_ENDPOINT` and `IDENTITY_HEADER` are set.
 
 ### Other Changes
+Added tests to validate ManagedIdentityCredential cases based on environment variables
```

Comment:
nit: Don't think this needs to be called out in a CHANGELOG as it doesn't impact customers

Reply:
yeah we can pull this out.